tests/test_vpc.rb

```diff
@@ -56,247 +49,251 @@ def test_vpc_create
 
   def test_vpc_destroy
     # create and test again
-    @vpc = Vpc.new(100)
-    @vpc.destroy
+    vpc = Vpc.new(100)
+    vpc.destroy
     refute(Vpc.enabled, 'VPC feature should have been disabled')
```

Comment:
Nit: We should not be disabling features anymore. I'm not suggesting you change this now but please be aware that there are other ways to achieve a destroy.